app/models/user.rb

```diff
@@ -378,6 +393,12 @@ def mpi_profile_birth_date
     mpi_profile.birth_date
   end
 
+  def mpi_profile_relationships
```

Comment:
Do we need this? Or can we just use mpi_profile.relationships?

Reply:
My thinking generally with these functions has been to eventually move direct access of `mpi_profile` itself either outside of User completely or with very restricted cases. Also I think in this case I wanted to make sure we returned an empty array on nil because I call `.map` on this function